sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/api/Azure.Monitor.OpenTelemetry.LiveMetrics.netstandard2.0.cs

```diff
@@ -1,3 +1,22 @@
+namespace Azure.Monitor.OpenTelemetry.LiveMetrics
+{
+    public partial class LiveMetricsExporterOptions : Azure.Core.ClientOptions
```

Comment:
Do we need to need to inherit ClientOptions?

Reply:
We could validate later too.

Reply:
Yes, this is required to initialize the RestClient.